students/saleta_sebastian/lesson_03_functions/fib.py

```diff
@@ -0,0 +1,9 @@
+def fib(n):
+    if n == 1 or n == 2:
+        return (1)
+    else:
+        return fib(n - 1) + fib(n - 2)
+
+
+n = int(input())
```

Comment:
Add if __name__ == '__main__' and main() function in wich you will get input from the user and print result